lib/status.lua

```diff
@@ -0,0 +1,18 @@
+local prefix = ARGV[1]
+local limit = tonumber(ARGV[2])
+local matches = {}
+local cursor = 0
+
+repeat
```

Comment:
Regarding replication is this operation also replicated in the replicas? if so I guess we should prevent that, it does not makes sense to replicate this operation, but I think that by default redis simply tries to replicate every command it gets in case it has side effects.

Reply:
This operation doesn't have writes therefore it doesn't get replayed in the replicas. It only replicates write commands.

Reply:
❤️